### PR TITLE
Update Requirements Docs for Consul Terraform Sync

### PR DESCRIPTION
**Pull request by lornasong**

### CODE REVIEW

**Review by eikenb — APPROVED** [review]
Nice linking to more info... read through it and everything sounded good.

File: website/pages/docs/nia/installation/requirements.mdx [hunk 1]
```diff
@@ -3,16 +3,60 @@ layout: docs
...
+
+To install a local Consul agent, refer to the [Getting Started: Install Consul Tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-install).
+
+### Run an Agent
+
+The Consul agent must be running in order to dynamically update network devices. To run the local Consul agent, you can run Consul in development mode which can be started with `consul agent -dev` for simplicity. For more details on running Consul agent, refer to the [Getting Started: Run the Consul Agent Tutorial](https://learn.hashicorp.com/tutorials/consul/get-started-agent?in=consul/getting-started).
+
+### Register Services
+
+Consul-Terraform-Sync monitors Consul catalog for service changes which lead to downstream changes to your network devices. Without services, your Consul-Terraform-Sync daemon will be operational but idle. You can register services with your Consul agent either by loading a service definition or by http api request.
```
Comment (findkim) [inline website/pages/docs/nia/installation/requirements.mdx:30]: "HTTP API request" to keep consistent with the usage in the next sentence

File: website/pages/docs/nia/installation/requirements.mdx [hunk 2]
```diff
@@ -3,16 +3,60 @@ layout: docs
...
+    "ID": "web",
+    "Name": "web",
+    "Address": "10.10.10.10",
+    "Port": 8000
+}' > payload.json
+
+$ curl --request PUT --data @payload.json http://localhost:8500/v1/agent/service/register
+```
+
+The above example registers a service named "web". You can optionally configure the "web" service with a [service block](/docs/nia/installation/configuration#service) if it has any non-default values. You can also have Consul-Terraform-Sync monitor the "web" service to execute a task and update network device(s) by configuring "web" in [`task.services`](https://deploy-preview-8881--consul-docs-preview.netlify.app/docs/nia/installation/configuration#services) of a task block.
```
Comment (findkim) [inline website/pages/docs/nia/installation/requirements.mdx:45]: Could you clarify where the service is registered to and what you're configuring "web" for? It's ambiguous when switching context between registering to Consul agent to then configuring for Sync.

For example adding:

> The above example registers a service named "web"

... to the Consul agent.

**Review by findkim — COMMENTED** [review]
I really like how this page ties together all the dependencies

File: website/pages/docs/nia/installation/requirements.mdx (hunk 2 above)
Comment (lornasong) [reply website/pages/docs/nia/installation/requirements.mdx:45]: Ah, I see what you mean about it being ambiguous. What do you think of something like:

The above example registers a service named "web" with your Consul agent. This represents a non-existent web service running at 10.10.10.10:8000. Your web service is now available for Consul-Terraform-Sync to consume. In Consul-Terraform-Sync, you can optionally configure the web service with a [service block](/docs/nia/installation/configuration#service) if it has any non-default values. You can also have Consul-Terraform-Sync monitor the web service to execute a task and update network device(s) by configuring "web" in [`task.services`](/docs/nia/installation/configuration#services) of a task block.

Comment (findkim) [reply website/pages/docs/nia/installation/requirements.mdx:45]: ✨🛳️

yeah that's really good! i like the added context. it walks readers through from Consul to Sync

Comment (lornasong) [reply website/pages/docs/nia/installation/requirements.mdx:45]: Whoops! Your comment made me realize I also included the netlify link in the docs 🤦

Comment (findkim) [reply website/pages/docs/nia/installation/requirements.mdx:45]: LOL i missed that. nice catch

Comment (lornasong) [reply website/pages/docs/nia/installation/requirements.mdx:45]: thanks @findkim 🙏

File: website/pages/docs/nia/installation/requirements.mdx (hunk 1 above)
Comment (lornasong) [reply website/pages/docs/nia/installation/requirements.mdx]: thanks for catching this!

**Pull request merged**